src/library/scala/collection/mutable/ResizableArray.scala

```diff
@@ -101,7 +101,7 @@ trait ResizableArray[A] extends IndexedSeq[A]
       if (newSize > Int.MaxValue) newSize = Int.MaxValue
 
       val newArray: Array[AnyRef] = new Array(newSize.toInt)
-      scala.compat.Platform.arraycopy(array, 0, newArray, 0, size0)
+      java.lang.System.arraycopy(array, 0, newArray, 0, size0)
```

Comment:
If you want to change all `Platform.arrayCopy` references, consider changning [line 120](https://github.com/scala/scala/pull/5516/files#diff-90ed9312d3d07483b0a1fc70860e0eb4L120) also